sphinx/about/whats-new.rst

```diff
@@ -1,9 +1,25 @@
 Version history
 ===============
 
-6.0.0-m1 (2018 September)
+6.0.0-m2 (2018 September)
 -------------------------
 
+Bio-Formats API changes:
```

Comment:
@sbesson @melissalinkert Is there any intention to address the comments in https://github.com/openmicroscopy/bioformats/pull/3189#issuecomment-409232132 before releasing?  Mentioning here because it's documenting the changes these comments were about.

Thanks,
Roger

Reply:
@rleigh-codelibre the goal here is primarily to document the changes that have been made and merged from the integration branc and will be released as a developmental milestone.
On our side, one of the next steps will be to start consuming, testing and validating this implementation against a few concrete use cases including generation of large datasets as well as analytical, export and conversion workflows.

None of the API changes in this intermediate milestone should be considered as final. A review of the various concerns and questions raised by during the initial OME-TIFF pyramid work including your comments will definitely be scheduled at a later milestone, before the final public 6.0.0 release.
Thanks for raising the issue, I will add a sentence to this version history file clarifying this point to make things unambiguous.